types/src/vm/modules.rs

```diff
@@ -31,6 +31,15 @@ impl AptosModuleExtension {
         }
     }
 
+    pub fn new_from_bytes(bytes: Bytes) -> Self {
+        let hash = sha3_256(&bytes);
+        Self {
+            bytes,
+            hash,
+            state_value_metadata: StateValueMetadata::none(),
```

Comment:
Is it OK to set `StateValueMetadata` to none here, @georgemitenkov?

Reply:
We should use one from the state view, none is not correct